Source/WebKit/UIProcess/API/glib/WebKitCookieManager.cpp

```diff
@@ -517,3 +517,106 @@ void webkit_cookie_manager_delete_all_cookies(WebKitCookieManager* manager)
     webkit_website_data_manager_clear(manager->priv->dataManager, WEBKIT_WEBSITE_DATA_COOKIES, 0, nullptr, nullptr, nullptr);
 }
 #endif
+
+/**
+ * webkit_cookie_manager_set_all_cookies:
+ * @cookie_manager: a #WebKitCookieManager
+ * @cookies: (element-type SoupCookie) (transfer none): a #GList of #SoupCookie to be added
+ * @operation: a #WebKitCookieManagerOperation
+ * @cancellable: (allow-none): a #GCancellable or %NULL to ignore
+ * @callback: (scope async): a #GAsyncReadyCallback to call when the request is satisfied
+ * @user_data: (closure): the data to pass to callback function
```

Comment:
```suggestion
 * @callback: (scope async): (closure): a #GAsyncReadyCallback to call when the request is satisfied
 * @user_data: the data to pass to callback function
```

The `(closure)` annotation is in the wrong place, a common mistake. If you remember where you copied this from, we should fix that place too (in a separate commit).

Reply:
Actually, don't worry about wherever you copied this from. We need to do a general audit because it's all over the place. I'll add it to my to-do list.

Reply:
Um, I amended my suggestion. This could be:

```
 * @callback: (scope async): (closure user_data): a #GAsyncReadyCallback to call when the request is satisfied
 * @user_data: the data to pass to callback function
```

But I think that's redundant, so we can just drop it.

Reply:
So I've confirmed that `(closure user_data)` is redundant, but in the spirit of the "explicit is better than implicit" rule I'm planning to move them to the correct place rather than remove them. Accordingly, my suggested change is exactly as in the previous comment:

```
 * @callback: (scope async): (closure user_data): a #GAsyncReadyCallback to call when the request is satisfied
 * @user_data: the data to pass to callback function
```

I'll handle auditing all preexisting uses, so all you need to worry about are the two cases added in this pull request.

Reply:
I actually wound up removing these (though adding an explicit one is not harmful, as long as it's on the `callback` parameter and not the `user_data` parameter).